helm/kubernetes-nginx-ingress-controller-chart/templates/controller-deployment.yaml

```diff
@@ -55,6 +55,8 @@ spec:
         name: sysctl
         securityContext:
           privileged: true
+          runAsUser: {{ .Values.initContainer.userID }}
+          runAsGroup: {{ .Values.initContainer.groupID }}
       containers:
```

Comment:
I am ok with this change, but I was thinking why not remove initContainer and apply `sysctl -w net.core.somaxconn=32768; sysctl -w net.ipv4.ip_local_port_range="1024 65535"` in the nodes as we did with max connection for Elastic?

Reply:
Probably because in future we can make ingress-controller optional or give customer a choice for ic. And different ic may require different improvements

Reply:
touché good point